src/LineChart/store/LineChartActions.ts

```diff
@@ -32,47 +32,79 @@ export const fetchData = (props: LineChartDataHandlerProps) => (dispatch: Dispat
             if (!props.fetchingData) {
                 dispatch({ type: TOGGLE_FETCHING_DATA, widgetID: props.friendlyId, fetchingData: true });
             }
+            const mixinSeries: Data.LineSeriesProps[] = props.series.filter(series => series.seriesType === "static");
+            const dynamicSeries = props.series.filter(series => series.seriesType === "dynamic");
 
-            Promise.all(props.series.map(series => {
-                const attributes = [ series.xValueAttribute, series.yValueAttribute ];
-                [ series.xValueSortAttribute, series.markerSizeAttribute ].forEach(attribute => {
-                    if (attribute) {
-                        attributes.push(attribute);
-                    }
-                });
-                const mxObject = props.mxObject as mendix.lib.MxObject;
-                const url = series.restUrl && generateRESTURL(mxObject, series.restUrl, props.restParameters);
+            Promise.all(dynamicSeries.map(dynSeries => {
+                const entityPath = dynSeries.seriesEntity.split("/");
```

Comment:
Should already be handled by fetchByXPath. Just passing in the seriesEntity should be sufficient, I hope ...